src/Integration/LocalServices/SolutionRuleSetsInformationProvider.cs

```diff
@@ -78,29 +78,31 @@ private IEnumerable<RuleSetDeclaration> GetProjectRuleSetsDeclarationsIterator(P
              * for C# and VB projects that use the new project system).
              */
 
-            bool found = false;
+            var declarations = new List<RuleSetDeclaration>();
```

Comment:
Simplification: using _yield_ didn't add anything here all and made debugging harder.